addons/binding/org.openhab.binding.onkyo/src/main/java/org/openhab/binding/onkyo/handler/OnkyoHandler.java

```diff
@@ -532,7 +535,11 @@ private void updateAlbumArt(String data) {
             try {
                 byte[] imgData = onkyoAlbumArt.getAlbumArt();
                 if (imgData != null && imgData.length > 0) {
-                    updateState(CHANNEL_ALBUM_ART, new RawType(imgData));
+                    String mimeType = onkyoAlbumArt.getAlbumArtMimeType();
+                    if (mimeType.isEmpty()) {
+                        mimeType = guessMimeTypeFromData(data.getBytes());
```

Comment:
What is the relation between `imgData` and `data`?

Reply:
Should you maybe use `imgData` here or are they the same?

Reply:
Great catch, it should be imgData